### PR TITLE
Added explore cc button on submit page

### LINKED ISSUE #225: [Feature] "Add 'Explore CC' Button to Header of /submit Page
## Problem
The absence of the "Explore CC" button on the [submit](https://resources.creativecommons.org/submit/) page limits user exploration of Creative Commons resources. 

## Description

Currently, the "Explore CC" button is available on most pages of the Creative Commons website, facilitating easy discovery and navigation of CC-licensed content.Integrating this button into the page header will enhance usability and encourage users to engage with CC content while contributing.

## Implementation
Just need to add CC Global Header component on the submit file.

## Screenshots
Explore CC button missing from the top-right corner.
Screenshot 2024-03-20 at 09-18-20 Submit a resource Creative Commons

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by praveen-rikhari**
## Fixes

Fixes #225 by @praveen-rikhari
- #225 

## Description

This PR resolves the issue of the missing "Explore CC" button on the /submit page by introducing a separate layout file, `submit.html`. Instead of manually including the content for `/sumbit` page, this approach streamlines the process by dedicating a specific layout file for the /submit page. By incorporating the Explore CC button within this layout, users can seamlessly navigate and explore CC resources while engaging with the submission process.

## Technical details

1. Created a new layout file named `submit.html` to handle the layout of the `/submit` page.
2. Integrated the "Explore CC" button within the "submit.html" layout file by including `header.html`, ensuring its visibility and accessibility to users visiting the /submit page.

#### Before :
Screenshot 2024-03-20 at 09-18-20 Submit a resource Creative Commons
#### After :
Screenshot 2024-03-21 at 13-34-18 Submit a resource Creative Commons(1)

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by praveen-rikhari** [comment]
Hi @possumbilities @TimidRobot Please have a look on my PR and let me know if any changes are requied. 
Thanks

File: docs/submission.html [hunk 1]
```diff
@@ -1,44 +1,5 @@
...
-<p>We encourage you to fork the project on GitHub, and submit resources by making pull requests on the project.</p> 
-<p>See <a href="https://github.com/creativecommons/cc-resource-archive">https://github.com/creativecommons/cc-resource-archive</a> for more info.</p>
-
-</div>
-
-</body>
-</html>
-
-
+---
```
Comment (possumbilities) [inline docs/submission.html:5]: You're missing an End of File line here.

Comment (praveen-rikhari) [reply docs/submission.html]: Fixed

File: docs/Gemfile.lock
Comment (possumbilities) [inline docs/Gemfile.lock:1]: you don't need to submit these changes in this file. Please keep the file, but revert the changes.

**Comment by praveen-rikhari** [comment]
Thanks @TimidRobot I appreciate the work and ready to do such more contributions
